README.md

```diff
@@ -35,6 +35,22 @@ should submit the patch on their own behalf.
 
 [The full list of individuals is available here.](individuals.md)
 
+## Howto:
+
+Issue a pull request:
+
+- include an edit to the `/individuals.md` file (put your name in alphabetic
+order) to include your own entry
+- include a new file in `/individuals/` (use [individuals/template.md](individuals/template.md))
+- format the commit comment as `individual: Add Newname` (prefix lowercase,
+colon, the rest capitalized)
+- indicate how you fare with regard to the points in the comments to the pull
+request (a template should be available soon)
+- never use trailing spaces (indentation using tabs only)
+- look if your submission passes the automated tests, if not, look at the bottom
+of the log to see what's wrong.
+- wait for your submission to be approved, fix the warnings you have received.
```

Comment:
nit: sorry, programmer habit: your list is inconsistent. here the items end with periods, but aren't capitalized
So either capitalize all the points, and add missing periods, or decide this is all "point form" and don't end with periods.

Reply:
That was intentional, since it ends the list **and** the paragraph as well. But I get your point and this is your file, plus you write English and I don't, so be it!

Accepted